tensorflow/python/ops/metrics_impl.py

```diff
@@ -187,9 +187,9 @@ def _safe_div(numerator, denominator, name):
   """
```

Comment:
Can you update the method description to "Divides two tensors element-wise, returning 0 ..."?